core/lib/web3_decl/src/client/network.rs

```diff
@@ -12,9 +12,10 @@ pub trait Network: 'static + Copy + Default + Sync + Send + fmt::Debug {
     fn metric_label(&self) -> String;
 }
 
-/// L1 (i.e., Ethereum) network.
+/// L1-compatible (e.g., Ethereum) network.
+/// Note, that this network does not have to be an L1. It can be any network that is compatible with Ethereum JSON-RPC.
 #[derive(Debug, Clone, Copy, Default)]
-pub struct L1(Option<L1ChainId>);
+pub struct L1(Option<SLChainId>);
```

Comment:
Maybe rename this struct something like `SL` and document that it represents a network for settlement? At least, this is how this marker is used in the `eth_client` crate.

Reply:
@slowli I keep this struct as `L1` because there are structs like `DynClient<L1>` which support Eth interface as it is on Ethereum and not for e.g. on L2. I am not sure what is better here, I rename it to `L1CompatibleSL`, does it look good?

Reply:
It's fine to keep it as-is, I suppose, especially given that types containing "eth" in their name are not renamed.